api/api/clubs/serializers.py

```diff
@@ -10,3 +10,21 @@ class Meta:
         extra_kwargs = {'id': {'read_only': True},
                         'slug': {'read_only': True}
                         }
+
+
+class ClubRegistrationRequestSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ClubRegistrationRequest
+        fields = ['id', 'user', 'club', 'fee_submitted', 'status', 'remark', 'updated_by']
+        extra_kwargs = {'id': {'read_only': True}}
+
+    def update(self, instance, validated_data):
+        updatable_fields = ['fee_submitted', 'status', 'remark', 'updated_by']
```

Comment:
updated_by should not be an updatable field. It should be populated on the backend. The user who sends the update request should be added to updated_by.

Reply:
`updated_by` is a `Foreign key` in `ClubRegistrationRequest` so is it possible ?

Reply:
`updated_by` should not be accepted as an input from the client. It should be set while processing the request in the view or serializer.